src/ert/run_models/base_run_model.py

```diff
@@ -336,24 +334,26 @@ def run_experiment(
     ) -> None:
         raise NotImplementedError("Method must be implemented by inheritors!")
 
-    def phaseCount(self) -> int:
-        return self._phase_count
+    def totalIterationsInExperiment(self) -> int:
```

Comment:
Suggest removing these functions and just operate directly on the property as there is no validation on the setter